### PR TITLE
bug(refs T31795): add slots for boilerplate

### PR DESCRIPTION
**Pull request by gruenbergerdemos**
**Ticket:** https://yaits.demos-deutschland.de/T31795

BoilerplateModal in DpEditor was replaced by slots and therefore the ui form rows in twig files were replaced directly with dpEditor and the needed slots. 
See also https://github.com/demos-europe/demosplan-core/pull/875

### CODE REVIEW

**Review by graupnerdemos — APPROVED** [review]
lgtm. I am a bit flustered by why we sometimes call the email body just that and sometimes call it email text, but that is of no consequence here.

**Review by ahmad-demos — DISMISSED** [review]
lgtm. codeReview & Test, thanks!

File: client/js/components/statement/DpBoilerPlateModal.vue [hunk 1]
```diff
@@ -123,7 +123,7 @@ export default {
     },
 
     insertBoilerPlate () {
-      this.$emit('insertBoilerPlate', this.textToBeAdded)
+      this.$emit('insert', this.textToBeAdded)
```
Comment (spiess-demos) [inline client/js/components/statement/DpBoilerPlateModal.vue:126]: When renaming the event, it should be changed in every occurrence. I find `insertBoilerPlate` also in some Vue templates.

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: client/js/components/statement/DpBoilerPlateModal.vue (hunk 1 above)
Comment (gruenbergerdemos) [reply client/js/components/statement/DpBoilerPlateModal.vue:126]: Yes, thanks, I forgot to commit and push these. Done in [3bd206d](https://github.com/demos-europe/demosplan-core/pull/969/commits/3bd206dbc5709e3adca7889557b6f4901619810e)

File: client/js/components/procedure/SegmentsBulkEdit/SegmentsBulkEdit.vue [hunk 2]
```diff
@@ -118,7 +118,7 @@
             id="addRecommendationTipTap"
             v-model="actions.addRecommendations.text"
             editor-id="recommendationText"
-            :toolbar-items="{
+            :toolbar-items="{https://github.com/demos-europe/demosplan-core/pull/969
```
[the remark this answers is not in the document]
Comment (gruenbergerdemos) [reply client/js/components/procedure/SegmentsBulkEdit/SegmentsBulkEdit.vue]: [a03bd28](https://github.com/demos-europe/demosplan-core/pull/969/commits/a03bd28a45679d4b14c33897f3a11786a310c3fb)

**Review by spiess-demos — APPROVED** [review]
lgtm! code review only

**Comment by graupnerdemos** [comment]
The SonarCloud fail is acceptable in this case and will be ignored.

**Pull request merged**